history/management/commands/compare_perf.py

```diff
@@ -1,58 +1,59 @@
 
 from django.core.management.base import BaseCommand
 from django.conf import settings
+from history.tools import get_fee_amount
 from history.models import Price, TradeRecommendation, PerformanceComp
 import datetime
 from django.db import transaction
+
+
 class Command(BaseCommand):
 
     help = 'compares market perf vs nn_perf'
 
     def handle(self, *args, **options):
-        #setup
+        # setup
         buffer_between_prediction_and_this_script_mins = datetime.datetime.now().minute % 10
-        granularity_mins = 10 #TODO: change me when granularity changes
+        granularity_mins = 10  # TODO: change me when granularity changes
         ticker = 'BTC_ETH'
 
-        #get data
-        date_of_timerange_we_care_about_predictions_start = datetime.datetime.now() - datetime.timedelta(seconds=((granularity_mins)*60+(60*(1+buffer_between_prediction_and_this_script_mins))))
-        date_of_timerange_we_care_about_predictions_end = datetime.datetime.now() - datetime.timedelta(seconds=((granularity_mins)*60))
-        tr_timerange_end = TradeRecommendation.objects.filter(symbol=ticker,created_on__gte=date_of_timerange_we_care_about_predictions_start,created_on__lte=date_of_timerange_we_care_about_predictions_end).order_by('-created_on').first().created_on
+        # get data
+        date_of_timerange_we_care_about_predictions_start = datetime.datetime.now() - datetime.timedelta(seconds=((granularity_mins) * 60 + (60 * (1 + buffer_between_prediction_and_this_script_mins))))
+        date_of_timerange_we_care_about_predictions_end = datetime.datetime.now() - datetime.timedelta(seconds=((granularity_mins) * 60))
+        tr_timerange_end = TradeRecommendation.objects.filter(symbol=ticker, created_on__gte=date_of_timerange_we_care_about_predictions_start, created_on__lte=date_of_timerange_we_care_about_predictions_end).order_by('-created_on').first().created_on
```

Comment:
How about we check if that kind of TradeRecommendation exists in the first place. If it doesn't we display an appropriate message. This could be done at the beginning of the management command.
This will throw an AttributeError if such TradeRecommendation does not exist.